sys/include/riot_stdio.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_stdio RIOT stdio
+ * @ingroup     sys
+ *
+ * @brief       stdio init/read/write functions
+ *
+ * @{
+ * @file
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ * @author      Anthony Merlino  <anthony@vergeaero.com>
+ */
+#ifndef RIOT_STDIO_H
+#define RIOT_STDIO_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef MODULE_UART_STDIO
```

Comment:
Are these includes necessary in the header? Shouldn't it be sufficient to include them in the used implementation .c?